src/UI/sd-ui/src/components/matchups/MatchupCard.css

```
@@ -292,6 +292,33 @@
+  color: var(--accent);
+  text-decoration: none;
+  transition: opacity 0.2s ease;
+}
+
+.game-preview-link:hover {
+  opacity: 0.8;
+  text-decoration: underline;
+}
+
+.game-preview-link:focus-visible {
+  outline: 2px solid currentColor;
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)